### PR TITLE
Rubocop fixes

### PR DESCRIPTION
**Pull request by SwatiTabib**

### CODE REVIEW

**Comment by SwatiTabib** [comment]
Out of 1083, 968 offences are resolved!

**Comment by pratyush23** [comment]
👍

File: lib/frameworks/capybara.rb [hunk 1]
```diff
@@ -115,14 +119,13 @@ def update_firefox_profile_with_certificates(profile, certificate_path, certific
     profile = Selenium::WebDriver::Firefox::Profile.new(profile_path)
 
     # Avoid Firefox certificate alerts
-    profile["security.default_personal_cert"] = 'Select Automatically'
+    profile['security.default_personal_cert'] = 'Select Automatically'
 
-    return profile
+    # return profile
```
Comment (ankitjain0105) [inline lib/frameworks/capybara.rb:124]: do we need to remove this ?

File: lib/frameworks/capybara.rb [hunk 2]
```diff
@@ -226,36 +225,51 @@ def register_mechanize_driver(opts)
...
     end
     :poltergeist
   end
 
+  def register_headless_chromium_driver(_opts)
+    app = Proc.new do
+      ['200', { 'Content-Type' => 'text/html' }, ['A barebones rack app.']]
+    end
+    Capybara.register_driver :headless_chromium do |app|
+      Capybara.app_host = 'http://www.bbc.co.uk'
```
Comment (ankitjain0105) [inline lib/frameworks/capybara.rb]: does it mean that it will run only on live ?

File: lib/frameworks/capybara.rb [hunk 3]
```diff
@@ -226,36 +225,51 @@ def register_mechanize_driver(opts)
...
+      caps = Selenium::WebDriver::Remote::Capabilities.chrome(
+        'chromeOptions' => {
+          'binary' => '/Users/tabibs01/Downloads/chrome-linux/chrome',
```
Comment (ankitjain0105) [inline lib/frameworks/capybara.rb]: Need to find the solution to make it generic location for e.g. keeping it in repo, thoughts ?

**Review by ankitjain0105 — CHANGES_REQUESTED** [review]
Just few things i have mentioned, please have a look.
Great work Swati...

File: lib/frameworks/capybara.rb (hunk 1 above)
Comment (SwatiTabib) [reply lib/frameworks/capybara.rb:124]: not now, I want to check if everything is ok on jenkins jobs before I remove

File: lib/frameworks/capybara.rb (hunk 2 above)
Comment (SwatiTabib) [reply lib/frameworks/capybara.rb]: this method shouldn't be a part of this PR I am taking out this

File: lib/frameworks/capybara.rb (hunk 3 above)
Comment (SwatiTabib) [reply lib/frameworks/capybara.rb]: see above

**Review by PriyaRam24 — COMMENTED** [review]
Looks good.Great work

**Comment by SwatiTabib** [comment]
@ankitjain0105 : removed chrome driver code for now.

**Review by ankitjain0105 — APPROVED** [review]

**Comment by ankitjain0105** [comment]
Thanks Swati :shipit:

**Pull request merged**